.github/workflows/ci.yml

```diff
@@ -359,42 +359,39 @@ jobs:
             **/surefire-reports/TEST-*.xml
           retention-days: ${{ env.TEST_REPORT_RETENTION_DAYS }}
 
+  build-test-matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # checkout all commits to be able to determine merge base for GIB
+      - name: Fetch base ref
+        run: |
+          [ "$GITHUB_REF_NAME" != master ] && git fetch --no-tags --prune --progress --no-recurse-submodules origin "$GITHUB_BASE_REF"
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'zulu'
+          java-version: 11
+          cache: 'maven'
+      - name: Maven install
+        run: |
+          export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
+          $RETRY $MAVEN install ${MAVEN_FAST_INSTALL} -pl '!:trino-docs,!:trino-server,!:trino-server-rpm'
```

Comment:
We should run `validate` here but for some reason it triggers the checkstyle plugin, which causes Maven to try to resolve dependencies and it fails. It does so even if there's `-Dair.check.skip-all` present. I'll look into it, it would surely save a few minutes and it's important to build the matrix as fast as possible and start other test jobs sooner.

Reply:
Looks like we'd need this first: https://github.com/airlift/airbase/pull/293

Reply:
I just released Airbase 120 with that change

Reply:
Yay, it worked! I had to also disable the check spi dependencies mojo from the trino-maven-plugin. See the last commit (I can squash it later).

Reply:
There are some other failures in tests that might be related to the upgrade, I'm looking into it.